packages/swap/src/helpers/orders/getMinAdaReceiveAfterSlippage.ts

```diff
@@ -0,0 +1,30 @@
+import {Balance, NumberLocale} from '@yoroi/types'
+import {Quantities} from '../../utils/quantities'
+import {asQuantity} from '../../utils/asQuantity'
+
+/**
+ * Calculate the minimum amount of ADA received after accounting for slippage.
+ *
+ * @param {Balance.Quantity} outputAmount - The amount of ADA or output currency.
+ * @param {number} slippagePercentage - The slippage percentage as a decimal (e.g., 3% as 0.03).
+ * @param {number} decimals - The number of decimal places for formatting the result.
+ * @param {NumberLocale} numberLocale - The locale information for formatting numbers.
+ *
+ * @returns {string} The minimum ADA amount received after slippage as a string with specified decimals.
+ */
+
+export const getMinAdaReceiveAfterSlippage = (
+  outputAmount: Balance.Quantity,
+  slippagePercentage: number,
+  decimals: number,
+  numberLocale: NumberLocale,
+): string => {
+  const slippageDecimal = slippagePercentage / 100
+  const result = Number(outputAmount) / (1 + slippageDecimal)
```

Comment:
`.Quantity` can hold big numbers, using `Number()` can lose precision, should it be `BigNumber` or Number is safe for this first parameter?, cuz if uncertain, `BN(outputAmount).div` ....

Reply:
@stackchain I've seen in the package we've not make use of BigNumber, using BigInt (native one) instead. Is there a reason behind that?

Reply:
https://github.com/facebook/hermes/issues/510

Reply:
have used `BigInt` as I saw that is used for the other functions in the package. Yes! seems like we can use it as well from what I see